app/models/widget.rb

```diff
@@ -1,2 +1,8 @@
 class Widget < ApplicationRecord
+  def do_widgety_things
+    Logger.info "Log noise!"
+    useless_var = "Never used"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.